<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WareHouseWidget</class>
 <widget class="QWidget" name="WareHouseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover
{
background-color: rgb(180, 85, 100);
color:white;
}

QPushButton:hover:pressed
{
background-color:rgb(180, 85, 100);
color:white;
}

QPushButton
{
background-color:rgb(84, 115, 135);
color:white;
font-size:12px;
width: 150px;
border-radius:2px;
spacing:2px;
padding: 0px 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="6,4">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widgetControl" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <widget class="QPushButton" name="pushButtonAddWareHouse">
             <property name="geometry">
              <rect>
               <x>5</x>
               <y>23</y>
               <width>150</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string> 添加仓库</string>
             </property>
             <property name="icon">
              <iconset resource="../../../res.qrc">
               <normaloff>:/image/image/addfile.png</normaloff>:/image/image/addfile.png</iconset>
             </property>
            </widget>
            <widget class="QLabel" name="labelSize">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>20</y>
               <width>16</width>
               <height>18</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidgetWareHouse">
            <property name="textElideMode">
             <enum>Qt::ElideRight</enum>
            </property>
            <column>
             <property name="text">
              <string>仓库列表</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidgetWareHouse">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabAddWareHouse">
        <attribute name="title">
         <string>添加仓库</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
background-color:rgb(180, 85, 100);
color:white;
font-size:14px;
font-weight:bold;
}</string>
             </property>
             <property name="text">
              <string> 基本信息</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>仓库名称：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditName">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>30</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>必填</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="labelAddType">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
background-color:rgb(180, 85, 100);
color:white;
font-size:14px;
font-weight:bold;
}</string>
             </property>
             <property name="text">
              <string> 此仓库出品的商品分类</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidgetAddMaterial">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover
{
background-color: rgb(180, 85, 100);
border-radius:5px;
color:white;
}

QPushButton:hover:pressed
{
background-color:rgb(180, 85, 100);
border-radius:5px;
color:white;
}

QPushButton
{
background-color:rgb(100, 85, 80);
color:white;
border-radius:5px;
height: 20px;
min-width: 30px;
spacing: 10px;
padding: 0px 0px;
}</string>
             </property>
             <property name="autoScrollMargin">
              <number>16</number>
             </property>
             <property name="resizeMode">
              <enum>QListView::Adjust</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAddSpType">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string> 添加商品分类</string>
             </property>
             <property name="icon">
              <iconset resource="../../../res.qrc">
               <normaloff>:/image/image/addfile.png</normaloff>:/image/image/addfile.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
background-color:rgb(180, 85, 100);
color:white;
font-size:14px;
font-weight:bold;
}</string>
             </property>
             <property name="text">
              <string> 此仓库出品的商品</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidgetAddExten">
             <property name="styleSheet">
              <string notr="true">QPushButton
{
background-color:transparent;
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAddSp">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string> 添加商品</string>
             </property>
             <property name="icon">
              <iconset resource="../../../res.qrc">
               <normaloff>:/image/image/addfile.png</normaloff>:/image/image/addfile.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButtonSave">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string> 保存</string>
               </property>
               <property name="icon">
                <iconset resource="../../../res.qrc">
                 <normaloff>:/image/image/save.png</normaloff>:/image/image/save.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonCancel">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string> 取消</string>
               </property>
               <property name="icon">
                <iconset resource="../../../res.qrc">
                 <normaloff>:/image/image/cancel.png</normaloff>:/image/image/cancel.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabWareHouseInfo">
        <attribute name="title">
         <string>仓库信息</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
background-color:rgb(180, 85, 100);
color:white;
font-size:14px;
font-weight:bold;
}</string>
             </property>
             <property name="text">
              <string> 基本信息</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string> 仓库名称：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditNameInfo">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>30</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>必填</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="labelTypeInfo">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
background-color:rgb(180, 85, 100);
color:white;
font-size:14px;
font-weight:bold;
}</string>
             </property>
             <property name="text">
              <string> 此仓库出品的商品分类</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidgetMaterialInfo">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="autoScrollMargin">
              <number>16</number>
             </property>
             <property name="resizeMode">
              <enum>QListView::Adjust</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAddSpTypeInfo">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string> 添加商品分类</string>
             </property>
             <property name="icon">
              <iconset resource="../../../res.qrc">
               <normaloff>:/image/image/addfile.png</normaloff>:/image/image/addfile.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
background-color:rgb(180, 85, 100);
color:white;
font-size:14px;
font-weight:bold;
}</string>
             </property>
             <property name="text">
              <string> 此仓库出品的商品</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidgetExtenInfo">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAddSpInfo">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string> 添加商品</string>
             </property>
             <property name="icon">
              <iconset resource="../../../res.qrc">
               <normaloff>:/image/image/addfile.png</normaloff>:/image/image/addfile.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButtonSaveInfo">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string> 保存</string>
               </property>
               <property name="icon">
                <iconset resource="../../../res.qrc">
                 <normaloff>:/image/image/save.png</normaloff>:/image/image/save.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonDelWareHouse">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string> 删除仓库</string>
               </property>
               <property name="icon">
                <iconset resource="../../../res.qrc">
                 <normaloff>:/image/image/删 除.png</normaloff>:/image/image/删 除.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonCancelInfo">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string> 取消</string>
               </property>
               <property name="icon">
                <iconset resource="../../../res.qrc">
                 <normaloff>:/image/image/cancel.png</normaloff>:/image/image/cancel.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res.qrc"/>
 </resources>
 <connections/>
</ui>
